src/python/WMCore/Services/Requests.py

```diff
@@ -236,7 +227,7 @@ def f():
 
         headers["Content-length"] = str(len(encoded_data))
 
-        assert isinstance(encoded_data, type('string')), \
+        assert isinstance(encoded_data, str), \
```

Comment:
please always use basestring instead of str, it covers both use case a string and unicode.